### DISCUSSION TITLE
General Repots

### DISCUSSION THREAD

**Discussion by TomAff** [moneymanagerex/moneymanagerex#5278] [Q&A]
There is is report => Categories => Summary

Is it possible to modify it that I get the Matrix
Jan | Feb | Mar | Apr ....... | total

and in the Sumary line
Cash in
Cash out
Profit / Loss
Balance

?

That tree view is what I use in excel at the moment.

**Comment by whalley** [comment]
Is the Categories/Monthly what you are asking for?

<img width="1229" alt="CleanShot 2022-10-31 at 15 02 15@2x" src="https://user-images.githubusercontent.com/487345/199040006-6136f9da-a730-4c6c-9fe8-973942435288.png">

**Comment by TomAff** [comment]
There is no tree view and it shows the last 12 month (I just have a look on the years)

**Comment by tactilis** [comment]
> There is no tree view and it shows the last 12 month (I just have a look on the years)

> That tree view is what I use in excel at the moment.

Can you post a screenshot of the way you organise this in Excel please

**Comment by TomAff** [comment]
![123](https://user-images.githubusercontent.com/46572503/199058072-5539045b-02cd-47ab-aebc-7977966334f1.jpg)

**Reply by vomikan** [reply]
There are minimum two ways:
1 create custom report (GRM)
2 Connecting Excel to SQLite via ODBC Driver

**Comment by TomAff** [comment]
1. I know, already tried it. But I'm not a pro so it didn't work. 
2. This is the way I do it. I use the sqlite3.exe and the alldata - view

Can I download the category=> summary report as grm?

**Reply by whalley** [reply]
> Can I download the category=> summary report as grm?

No. The baked in reports are not written using GRM.
